src/app/file-explorer.js

```diff
@@ -20,12 +20,9 @@ var css = csjs`
     background-color  : white;
   }
   .remove             {
-    align-self        : center;
-    padding-left      : 10px;
+    float             : right;
```

Comment:
In that case, we don't want to `align-items: start` or `end`, we just want to put the `remove` icon at the end. can  `flex`  able to handle that?